src/components/Sidebar.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { storeCustomers } from "../data";
+import { CustomerConsumer } from "./CustomerContext";
+
+export default function Sidebar() {
+  return (
+    <CustomerConsumer>
+      {(ctx) => {
+        function getSelectedCustomer() {
+         let selectedCustomer = document.getElementById("storeCustomers")
```

Comment:
uh no, in 99% cases you don't need to use `document` reference in React.

PS you can move the definition of such a helper function up to line 7.
otherwise you define this function on EVERY render

Reply:
please google for it - how to use <select> and <option>s with React state properly

Reply:
I would expect that e.g. there's an `event` object passed into this handler function, that holds information about selected option. But just guessing

Reply:
in the `onChange` function you should be getting such an event data:

```
handleChange = (event) => {
  this.setState({ [event.target.name]: event.target.value });
}
```

so just use it this way `ctx.setCustomer(event.target.value)`